tornado/iostream.py

```diff
@@ -792,7 +797,10 @@ def _find_read_pos(self):
             # _consume().
             if self._read_buffer:
                 while True:
-                    loc = self._read_buffer[0].find(self._read_delimiter)
+                    try:
+                        loc = self._read_buffer[0].find(self._read_delimiter)
+                    except AttributeError:  # might be a memoryview
+                        loc = self._read_buffer[0].tobytes().find(self._read_delimiter)
```

Comment:
This bit is unfortunate.  There are times when we *need* the object to be a bytes object.  There might be ways to get the bytes object without doing the explicit conversion.